graspy/inference/latent_distribution_test.py

```diff
@@ -81,8 +91,11 @@ def __init__(self, n_components=None, method="mgc", pass_graph=True):
         super().__init__(embedding="ase", n_components=n_components, pass_graph=pass_graph)
         self.method = method
         self.symmetry = None
+        self.n_bootstraps = n_bootstraps
 
     def _k_sample_transform(self, x, y):
+        if x.shape[0] != y.shape[0]:
```

Comment:
good call. Let's just make it say something about when one of the graphs is small and the number of vertices is not the same. the n1/n2 stuff is not super clear

Reply:
don't use logging.warning either. Raise a UserWarning

Reply:
Don't do this in the k sample function. your error gets thrown n_boostraps times

Reply:
Why do we need our own k sample transform? MGC has one?